wgpu/gui/offscreen.py

```diff
@@ -76,11 +76,6 @@ def draw(self):
 
 def call_later(delay, callback, *args):
     loop = asyncio.get_event_loop_policy().get_event_loop()
-    # for the offscreen canvas, we prevent new frames and callbacks
-    # from being queued while the loop is running. this avoids
-    # callbacks from one visualization leaking into the next.
-    if loop.is_running():
-        return
     loop.call_later(delay, callback, *args)
```

Comment:
Oh, this un-does #322, so this is probably not a good idea ...  

I was under the impression that this was a leftover and technically not needed because we cancel all tasks below. But these are times, not tasks.